### PR TITLE
Mention that you do not need to set up the debugger if you are using the Inspector SDK on Android and iOS

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…using-visual-debuggers**](https://docs-oi5s0gzw0.vercel.app/explore-tracking-plan/start-using-visual-debuggers) | [**…-using-visual-debugger**](https://docs-oi5s0gzw0.vercel.app/implementation/start-using-visual-debugger) |
|:-:|:-:|
|<a href="https://docs-oi5s0gzw0.vercel.app/explore-tracking-plan/start-using-visual-debuggers">Screenshot of /explore-tracking-plan/start-using-visual-debuggers</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/772USkfZUSmW3ynfwnAO8Tx3qbY_.png">(view full size)</a> |<a href="https://docs-oi5s0gzw0.vercel.app/implementation/start-using-visual-debugger">Screenshot of /implementation/start-using-visual-debugger</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/yykn5DwuVh4Y7xocqjW0kPyYl2w_.png">(view full size)</a> |







Commit f38330a791113008e8528f0eccc11c6a343953dc (https://docs-oi5s0gzw0.vercel.app).

File: pages/explore-tracking-plan/start-using-visual-debuggers.mdx [hunk 1]
```diff
@@ -31,6 +31,8 @@ Example: `https://example.com/checkout?avo_debug=1`.
 
 ### Accessing mobile debuggers
 
+> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspectos SDKs.
```
Comment (alex-tpom6oh) [inline pages/explore-tracking-plan/start-using-visual-debuggers.mdx:34]:
```suggestion
> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs.
```

File: pages/implementation/start-using-visual-debugger.mdx [hunk 2]
```diff
@@ -31,6 +31,8 @@ To access it in production add the `?avo_debug=1` query parameter to the URL.
 
 ### Setting up the mobile debuggers
 
+> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspectos SDKs.
```
Comment (alex-tpom6oh) [inline pages/implementation/start-using-visual-debugger.mdx:34]:
```suggestion
> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs.
```

File: pages/explore-tracking-plan/start-using-visual-debuggers.mdx [hunk 3]
```diff
@@ -31,6 +31,8 @@ Example: `https://example.com/checkout?avo_debug=1`.
 
 ### Accessing mobile debuggers
 
+> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs.
```
Comment (logason) [inline pages/explore-tracking-plan/start-using-visual-debuggers.mdx:34]: I feel like this is telling me the setup is optional, and I will only need to do it if I want the visual debuggers? Is that the message we're trying to deliver here? Or is this setup unnecessary if I already have the Inspector SDK initialized? If that's the case, we could maybe frame it like this:

```suggestion
> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Inspector. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Inspector.
```

File: pages/implementation/start-using-visual-debugger.mdx [hunk 4]
```diff
@@ -31,6 +31,8 @@ To access it in production add the `?avo_debug=1` query parameter to the URL.
 
 ### Setting up the mobile debuggers
 
+> If you are using the Inspector SDK on iOS and Android the Visual Debugger is bundled with it, you do not need to do this setup! See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs.
```
Comment (alex-tpom6oh) [inline pages/implementation/start-using-visual-debugger.mdx:34]:
```suggestion
> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Debugger. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Inspector.
```

File: pages/explore-tracking-plan/start-using-visual-debuggers.mdx (hunk 3 above)
Comment (alex-tpom6oh) [reply pages/explore-tracking-plan/start-using-visual-debuggers.mdx:34]: Makes sense!

File: pages/explore-tracking-plan/start-using-visual-debuggers.mdx [hunk 5]
```diff
@@ -31,6 +31,8 @@ Example: `https://example.com/checkout?avo_debug=1`.
 
 ### Accessing mobile debuggers
 
+> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Debugger. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Inspector.
```
Comment (alex-tpom6oh) [inline pages/explore-tracking-plan/start-using-visual-debuggers.mdx:34]:
```suggestion
> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Debugger. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Debugger.
```

File: pages/implementation/start-using-visual-debugger.mdx [hunk 6]
```diff
@@ -31,6 +31,8 @@ To access it in production add the `?avo_debug=1` query parameter to the URL.
 
 ### Setting up the mobile debuggers
 
+> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Debugger. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Inspector.
```
Comment (alex-tpom6oh) [inline pages/implementation/start-using-visual-debugger.mdx:34]:
```suggestion
> If you have already initialized the Inspector SDK on iOS and Android, no further action is required to enable the Visual Debugger. See `showVisualInspector` and `hideVisualInspector` methods of the Inspector SDKs to start using the Visual Debugger.
```

**Pull request merged**